Base/res/apps/Playground.af

```diff
@@ -0,0 +1,4 @@
+[App]
```

Comment:
I know I added some older commits, but I don't really know/understand what happened And my git is not that great to fix that, so  maybe so help would be appreciated. :)

Reply:
To fix that, you probably want to rebase on top of upstream master, `git rebase master` if your master branch is up-to-date.

Reply:
Should be fixed now. Thanks. :)